cli/src/test/java/de/jplag/cli/OldNewRootDirectoriesArgumentTest.java

```diff
@@ -2,54 +2,67 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+import java.io.IOException;
+
 import org.junit.jupiter.api.Test;
 
-class OldNewRootDirectoriesArgumentTest extends CommandLineInterfaceTest {
+import de.jplag.cli.test.CliArg;
+import de.jplag.cli.test.CliArgBuilder;
+import de.jplag.cli.test.CliTest;
+import de.jplag.exceptions.ExitException;
+import de.jplag.options.JPlagOptions;
+
+class OldNewRootDirectoriesArgumentTest extends CliTest {
     @Test
-    void testNoRootDirectories() throws CliException {
-        buildOptionsFromCLI(arguments());
+    void testNoRootDirectories() throws ExitException, IOException {
+        JPlagOptions options = runCliForOptions();
 
         assertEquals(0, options.submissionDirectories().size());
         assertEquals(0, options.oldSubmissionDirectories().size());
     }
 
     @Test
-    void testTwoRootDirectoryArguments() throws CliException {
-        buildOptionsFromCLI(arguments().rootDirectory("root1", "root2"));
+    void testTwoRootDirectoryArguments() throws ExitException, IOException {
+        JPlagOptions options = runCliForOptions(args -> args.with(CliArg.SUBMISSION_DIRECTORIES, new String[] {"root1", "root2"}));
```

Comment:
Maybe also extract this array into a variable and then use its length in the assert below
Same with all other arrays in this test class